.github/workflows/test.yml

```diff
@@ -119,6 +119,7 @@ jobs:
           uv pip install -e ".[test,${{ matrix.model.name }}]" --resolution=${{ matrix.version.resolution }} --system
 
       - name: Run ${{ matrix.model.test_path }} tests
+        if: ${{ !contains(matrix.model.name, 'fairchem') || github.event.pull_request.head.repo.fork == false }}
```

Comment:
The tests show as passing with this change rather than not showing due to being skipped. This isn't ideal.

Reply:
How about we set an env variable with the repo in the action and then skip in `pytest` if the repo isn't TorchSim?

Something like:

```yaml
      - name: Run tests
        env:
          REPO: ${{ github.repository }}
```

```python

@pytest.mark.skipif(
    os.getenv("REPO") != os.getenv("TorchSim/torch-sim"), # or something
    reason="Skipping tests for forked PR"
)
def test_fairchem():
```

Reply:
functionally that's the same as this? they will just be skipped in the action and the action will be green? All the fairchem tests rely on HF for the models they use

Reply:
show as passing here is about Github UX not pytest